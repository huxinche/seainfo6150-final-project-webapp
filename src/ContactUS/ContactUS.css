.contact-us h1,.contact-us h2, .contact-us h3{
    font-family: 'Oswald', cursive;
}

.contact-us p{
    margin: 0px;
}

.contact-us .text-underline{
    text-decoration: underline;
    color: darkcyan;
}