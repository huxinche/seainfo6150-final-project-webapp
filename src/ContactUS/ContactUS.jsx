import React from 'react';
import Banner from '../Components/Common/Banner/Banner';
import ContactUSBackground from '../images/ContactUsBackground.jpeg';
import './ContactUS.css';
const ContactUS = () => {
    return <> <Banner image={ContactUSBackground}
        title={"Contact US"}
    />
        <div className="container">
            <div className="contact-us">
                <h1>Contact Us</h1>

                <h2>Prospective Graduate Students and Applicants</h2>
                <p>We invite you to direct all questions about graduate admission, including academic requirements, admission eligibility, admission application requirements and status updates to the appropriate school and college.</p>

                <h2>Faculty, Staff, and Current Students</h2>
                <p>Please contact or visit the Associate Provost for Graduate Affairs with questions related to University policy, diversity and inclusion initiatives, the creation or revision of academic programs, graduate student resources, assessment and learning outcomes, and graduate enrollment management.</p>

                <h3>Office Location</h3>
                <p>Boston University</p>
                <p>Associate Provost for Graduate Affairs</p>
                <p>Office of the Provost</p>
                <p className="text-underline">University Way, 6th Floor</p>
                <p className="text-underline">Boston, MA 02411</p>


                <h3>Phone</h3>
                <p><b>Administrative Office:</b> <span className="text-underline">617-353-2230</span></p>

                <h3>Email</h3>
                <p><span className="text-underline">apgrad@bu.edu</span></p>
                <br />
                <br />
            </div>
        </div>

    </>
}

ContactUS.propTypes = {
}


export default ContactUS;