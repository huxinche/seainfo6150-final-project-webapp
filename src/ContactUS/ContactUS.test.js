import React from "react";
import ContactUS from "./ContactUS";

it("renders the ContactUS component correctly", () => {
    
    const { container } = render(<ContactUS />);
    expect(container).toMatchSnapshot();
})