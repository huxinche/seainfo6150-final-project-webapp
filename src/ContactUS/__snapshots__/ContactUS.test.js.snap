// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the ContactUS component correctly 1`] = `
<div>
   
  <div
    class="banner"
  >
    <div>
      <picture
        class="banner-img"
      >
        <img
          alt="Contact US"
          class="banner-img"
          src="ContactUsBackground.jpeg"
        />
      </picture>
    </div>
    <div
      class="banner-content"
    >
      <div>
        <h1
          class="text-title"
        >
          Contact US
        </h1>
        <p
          class="text-title"
        />
      </div>
    </div>
  </div>
  <div
    class="container"
  >
    <div
      class="contact-us"
    >
      <h1>
        Contact Us
      </h1>
      <h2>
        Prospective Graduate Students and Applicants
      </h2>
      <p>
        We invite you to direct all questions about graduate admission, including academic requirements, admission eligibility, admission application requirements and status updates to the appropriate school and college.
      </p>
      <h2>
        Faculty, Staff, and Current Students
      </h2>
      <p>
        Please contact or visit the Associate Provost for Graduate Affairs with questions related to University policy, diversity and inclusion initiatives, the creation or revision of academic programs, graduate student resources, assessment and learning outcomes, and graduate enrollment management.
      </p>
      <h3>
        Office Location
      </h3>
      <p>
        Boston University
      </p>
      <p>
        Associate Provost for Graduate Affairs
      </p>
      <p>
        Office of the Provost
      </p>
      <p
        class="text-underline"
      >
        University Way, 6th Floor
      </p>
      <p
        class="text-underline"
      >
        Boston, MA 02411
      </p>
      <h3>
        Phone
      </h3>
      <p>
        <b>
          Administrative Office:
        </b>
         
        <span
          class="text-underline"
        >
          617-353-2230
        </span>
      </p>
      <h3>
        Email
      </h3>
      <p>
        <span
          class="text-underline"
        >
          apgrad@bu.edu
        </span>
      </p>
      <br />
      <br />
    </div>
  </div>
</div>
`;
